tembo-operator/tests/integration_tests.rs

```diff
@@ -4369,19 +4369,19 @@ CREATE EVENT TRIGGER pgrst_watch
         assert!(service_items.is_empty());
         // should be no Services
 
-        // ingress must be gone
+        // There should be 1 IngressRoute left (metrics)
```

Comment:
These assertions were checking for 0 `IngressRoute`s and 0 `IngressRouteTCP`s. They were being deleted and re-created before introducing this fix. We expect 1 of each to be present.